pkg/metricscollector/opentelemetry.go

```diff
@@ -246,3 +246,28 @@ func getScalerMeasurementOption(namespace string, scaledObject string, scaler st
 		attribute.Key("metric").String(metric),
 	)
 }
+
+func (o *OtelMetrics) RecordScalerPaused(namespace string, scaledObject string, active bool) {
+	activeVal := 0
+	if active {
+		activeVal = 1
+	}
+
+	opt := api.WithAttributes(
+		attribute.Key("namespace").String(namespace),
+		attribute.Key("scaledObject").String(scaledObject),
+	)
+
+	cback := func(ctx context.Context, obsrv api.Float64Observer) error {
+		obsrv.Observe(float64(activeVal), opt)
+		return nil
+	}
+	_, err := meter.Float64ObservableGauge(
+		"keda.scaledobject.paused.total",
+		api.WithDescription("Total number of scaled objects currently in a paused state"),
+		api.WithFloat64Callback(cback),
+	)
+	if err != nil {
+		fmt.Println("failed to register scaled object paused total metric")
```

Comment:
don't use Println here

Reply:
@zroubalik could you please update the code with the new logging format so i can replicate and resolve this conversation.

Reply:
https://github.com/kedacore/keda/pull/5095